packages/block-library/src/file/index.php

```diff
@@ -107,13 +63,11 @@ function register_block_core_file() {
 		)
 	);
 
-	if ( defined( 'IS_GUTENBERG_PLUGIN' ) && IS_GUTENBERG_PLUGIN ) {
-		gutenberg_register_module(
-			'@wordpress/block-library/file-block',
-			gutenberg_url( '/build/interactivity/file.min.js' ),
-			array( '@wordpress/interactivity' ),
-			defined( 'GUTENBERG_VERSION' ) ? GUTENBERG_VERSION : get_bloginfo( 'version' )
-		);
-	}
+	wp_register_script_module(
+		'@wordpress/block-library/file-block',
+		gutenberg_url( '/build/interactivity/file.min.js' ),
```

Comment:
Hey folks, this is breaking the package update in Core. The block library php files are copied as is into WordPress Core. Meaning we should ensure they work there without changes.

 These Gutenberg specific function calls are there for not something we should be using. I see that the previous versions had these being the IS_GUTENBERG_PLUGIN to solve that.

Reply:
Opsss. Our bad. I'll open a PR right away!

Reply:
Do you want us to release a 17.6.1 RC with the fix?

cc: @c4rl0sbr4v0

Reply:
We will do a RC2. We don't need a 17.6.1 yet, as 17.6 won't be released until next week 😅

Reply:
There you go: 

- https://github.com/WordPress/gutenberg/pull/58242